web/templates/confirm_dialog/confirm_revoke_invite.hbs

```diff
@@ -1,5 +1,9 @@
 {{#if is_multiuse}}
+{{#if referred_by}}
 <p>{{#tr}}Are you sure you want to revoke this invitation link created by <strong>{referred_by}</strong>?{{/tr}}</p>
 {{else}}
+<p>{{#tr}}Are you sure you want to revoke this invitation link ?{{/tr}}</p>
```

Comment:
There shouldn't be a space before the `?` here; I'll just fix that before merging.

Also, I wonder if it'd be better to make the check "If the referrer is anyone other than me"?